front/src/common/Map/Settings/MapSettings.js

```diff
@@ -18,70 +19,57 @@ export default function MapSettings(props) {
     setShowSettings((prevState) => !prevState);
   };
 
-  if (active) {
-    return (
-      <div className={`map-modal${active ? ' active' : ''}`}>
-        <div className="h2">{t('map-settings:mapSettings')}</div>
-        <MapSettingsTrackSources />
-        <div className="my-1" />
-        <MapSettingsMapStyle />
-        <div className="my-1" />
-        <MapSettingsBackgroundSwitches />
-        <div
-          className="mb-1 mt-3 border-bottom d-flex align-items-center sub-section-title"
-          onClick={() => toogleShowSettings(setShowSettingsSignals)}
-          role="button"
-          tabIndex={0}
-        >
-          {t('map-settings:signalisation')}
-          <i
-            className={`${showSettingsSignals ? 'open' : ''} icons-arrow-down icons-size-20px`}
-            aria-hidden="true"
-          />
-        </div>
-        {showSettingsSignals && <MapSettingsSignals />}
-        <div
-          className="mb-1 mt-3 border-bottom d-flex align-items-center sub-section-title"
-          onClick={() => toogleShowSettings(setShowSettingsLayers)}
-          role="button"
-          tabIndex={0}
-        >
-          {t('map-settings:layers')}
-          <i
-            className={`${showSettingsLayers ? 'open' : ''} icons-arrow-down icons-size-20px`}
-            aria-hidden="true"
-          />
-        </div>
-        {showSettingsLayers && <MapSettingsLayers />}
-        <div
-          className="mb-1 mt-3 border-bottom d-flex align-items-center sub-section-title"
-          onClick={() => toogleShowSettings(setShowSettingsSpeedLimits)}
-          role="button"
-          tabIndex={0}
-        >
-          {t('map-settings:speedlimits')}
-          <i
-            className={`${showSettingsSpeedLimits ? 'open' : ''} icons-arrow-down icons-size-20px`}
-            aria-hidden="true"
-          />
-        </div>
-        {showSettingsSpeedLimits && <MapSettingsSpeedLimits />}
-        <div className="mt-2 d-flex flex-row-reverse w-100">
-          <button className="btn btn-secondary btn-sm" type="button" onClick={toggleMapSettings}>
-            {t('translation:common.close')}
-          </button>
-        </div>
+  return (
+    <div className="map-modal">
+      <HearderPopUp onClick={toggleMapSettings} title={t('map-settings:mapSettings')} />
```

Comment:
It would be clearer if the function name was closeMapSettingsPopup

Reply:
Done